docs/about/doc_overview.adoc

```diff
@@ -124,11 +128,18 @@ xref:{rootdir}/mp/guides/overview.adoc[MP Guides]
 .Training and Certification
 [icon=model_training]
 --
-Learn:  
+ 
+
+Training:  
  https://learn.oracle.com/ols/learning-path/become-a-helidon-microservices-developer-professional/88258/114512[Become a Helidon Microservices Developer Professional]
  
 Certification: 
   https://learn.oracle.com/ols/learning-path/become-a-helidon-microservices-developer-professional/88258/114512[Helidon Microservices Developer Professional (1Z0-1113)]
+  
+New to Helidon? Check out https://a.co/d/0niQnwC[Beginning Helidon] to learn more about building cloud-native microservices with Helidon.  
```

Comment:
I have another PR for book integration https://github.com/helidon-io/helidon/pull/7577
Maybe reuse mine?